doc/btp_gap.txt

```diff
@@ -891,3 +973,89 @@ Events:
+
+        Opcode 0x94 - BIS Data Path Setup Event
+
+                Controller Index:       <controller id>
+                Event parameters:       Address_Type (1 octet)
+                                        Address (6 octets)
+                                        BIS_ID (1 octet)
+
+                This event can be sent after IUT has been setup the data path for the specific BIS.
+                The parameter BIS_ID is corresponding to specific BIS.
+                The BIS data path should be setup when the BIG sync is established or BIG has
+                been created. And the BIS data path setup event indicates the successful
+                configuration of data transmission path for a specific Broadcast Isochronous
+                Stream (BIS).
+
+        Opcode 0x95 - BIS Stream Received Event
+
+                Controller Index:       <controller id>
+                Event parameters:       Address_Type (1 octet)
+                                        Address (6 octets)
+                                        BIS_ID (1 octet)
+                                        Flags (1 octet)
+                                        Timestamp (4 octets)
+                                        Sequence_Number (2 octets)
+                                        Data Length (1 octet)
+                                        Data (variable)
+
+                This event can be sent after IUT received the stream data from the specific BIS.
+
+                Available Flags:
+                        Bit 0: The packet is valid
+                        Bit 1: The packet may possibly contain errors
+                        Bit 2: The Packet was lost
+                        Bit 3: The Timestamp parameter is valid
+
+        Opcode 0x96 - Periodic BIGInfo Event
+
+                Controller Index:       <controller id>
+                Event parameters:       Address_Type (1 octet)
+                                        Address (6 octets)
+                                        Sync Handle (2 octets)
+                                        SID (1 octet)
+                                        Num_BIS (1 octet)
+                                        NSE (1 octet)
+                                        ISO_Interval (2 octets)
+                                        BN (1 octet)
+                                        PTO (1 octet)
+                                        IRC (1 octet)
+                                        Max_PDU (2 octets)
+                                        SDU_Interval (4 octets)
+                                        Max_SDU (2 octets)
+                                        PHY (1 octet)
+                                        Framing (1 octet)
+                                        Encryption (1 octet)
+
+                This event can be sent after IUT received the periodic BIGInfo Advertising Report
+                on established sync.
+
+                Available Encryption:
```

Comment:
You provide the possible values of the Encryption field here, but not e.g. the Framing or PHY fields